maps/example/example_areas.dm

```diff
@@ -33,23 +32,18 @@
 	arrival_sound = null
 	lift_announce_str = null
 
-/area/turbolift/alert_on_fall(var/mob/living/carbon/human/H)
-	if(H.client && SSpersistence.elevator_fall_shifts > 0)
```

Comment:
Why remove this?

Reply:
I don't think this needs to be here permanently on debug map.